build-common.properties

```diff
@@ -62,6 +62,7 @@ hamcrest.lib=${framework.loc}/tests/hamcrest-core-1.3.jar
 
 checker.loc=${checkerframework}/checker
 checker.lib=${checker.loc}/dist/checker.jar
+checker.lib.tmp=${checker.loc}/dist/checker-tmp.jar
```

Comment:
I am not sure about this one. 
It was added in https://github.com/Bohdankm22/checker-framework/commit/edc02b4fa2e2a6507d814d666337db4fc4fc7122
@mernst could you please comment?